submissions/Liddell777/hooli-style-popup/style.css

```diff
@@ -0,0 +1,277 @@
+}
+
+input::-ms-clear {
+    display: none;
+}
+
+button {
+    cursor: pointer;
+}
+
+button::-moz-focus-inner {
+    padding: 0;
+    border: 0;
+}
+
+a, a:visited, a:hover  {
+    text-decoration: none;
+}
+
+ul {
+    list-style: none;
+}
+
+img {
+    vertical-align: top;
+    max-width: 100%;
+}
+main {
+    background: 
+    url(../img/bg5.gif) center no-repeat;
+    background-size: 500px;
+}
+
+
+header {
+    width: 100vw;
+    height: 80px;
+} 
+
+.nav {
+    max-width: 90vw;
+    margin: auto;
+    padding: 25px 0 0;
+}
+
+.nav__menu {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.menu__item {
+    margin: 0 30px 0 0;
+    width: 42px;
+    height: 40px;
+}
+
+.menu__item img {
+    object-fit: contain;
+    width: 100%;
+    height: 100%;
+}
+.menu__link-user img {
+    border-radius: 50%;
+}
+
+.menu {
+    position: relative;
+}
+
+.menu__input {
+    position: absolute;
+    width: 30px;
+    height: 30px;
+    cursor: pointer;
+    opacity: 0;
+}
+
+.menu__label {
+    position: relative;
+    cursor: pointer;
+}
+
+.mainpopup {
+    margin: 25px 100px 0 -200px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 345px;
+    height: 460px;
+    overflow-y: scroll;
+    flex-wrap: wrap;
+    background: #fbfbfb;
+    border: 5px solid black;
+    border-radius: 4px;
+}
+.mainpopup::before {
+    content: "";
+    position: absolute;
+    top: 40px;
+    left: 50%;
+    transform: translate(-45%, 0);
+    width: 0;
+    height: 0;
+    border-style: solid;
+    border-width: 0 15px 20px 15px;
+    border-color: transparent transparent #000000 transparent;
+    display: none;
+}
+
+.mainpopup::-webkit-scrollbar {
+    width: 5px;
+}
+
+.mainpopup::-webkit-scrollbar-track {
+    background: #787a7ad8;
+}
+
+.mainpopup::-webkit-scrollbar-thumb {
+    background: #ec8179;
+}
+
+.mainpopup::-webkit-scrollbar-thumb:hover {
+    background: black;
+}
+
+.mainpopup__link {
+    width: 50px;
+    margin: 8px;
+    padding: 15px;
+    height: 50px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    box-sizing: content-box;
+    border-radius: 4px;
+}
+
+.mainpopup__link:hover {
+    border: 2px solid black;
+    padding: 10px;
+}
+.mainpopup__link:focus {
+    background-color: #a19999b9;
+}
+
+.mainpopup__link img {
+    object-fit: contain;
+    width: 70%;
+    height: 70%;
+}
+
+.mainpopup__link p {
+    margin: 10px 0 0;
+    color: black;
+}
+
+.mainpopup__item-input {
+    position: absolute;
+    width: 20px;
+    height: 10px;
+    cursor: pointer;
+    opacity: 0;
+    top: 70%;
+}
+
+.mainpopup__item-label {
+    position: relative;
+    cursor: pointer;
+    flex: 0 0 90%;
+    text-align: center;
+    border: 5px solid black;
+    border-radius: 4px;
+    padding: 3px 0;
+    margin: 40px 0;
+}
+
+.mainpopup__item-label:hover {
+    background-color: #eb665d;
+}
+
+.mainpopup__item-label:before {
+    content: "More";
```

Comment:
This text not accessible for screen readers. It must be in HTML markup.

Reply:
I tried to replace it with such a construction: 
 label for="main-popup__input" class="main-popup__item-label">More</label
CSS: 
.main-popup__item-label:hover {
    font-size: 0px;
    background-color: #eb665d;
}
.main-popup__item-label:hover::after {
    font-size: 16px;
    content: "less";
    background-color: #eb665d;
}

but I don't know how it right does it in my version code structures, maybe could u help me.. what I need change? thanks